doc/classes/Node.xml

```diff
@@ -421,6 +421,12 @@
 				Returns the node's [Viewport].
 			</description>
 		</method>
+		<method name="get_window" qualifiers="const">
+			<return type="Window" />
+			<description>
+				Returns the [Window] that contains this node. If the node is in the main window, this is equivalent to getting the root node ([code]get_tree().get_root()[/code]).
```

Comment:
A node can be within multiple nested windows. It should be clarified as in the following suggestion:

    Returns the _nearest_ [Window] that contains this node.

Reply:
Seems like something obvious, I'm not sure if it's necessary to clarify it.